pkg/entrypoint/entrypoint.go

```diff
@@ -55,6 +55,10 @@ func StartAllServicesWithLogger(logger *zap.Logger, zapConfigs *zap.Config) *Ser
 	return StartAllServices()
 }
 
+func InitializeLogger(logger *zap.Logger, zapConfigs *zap.Config) {
+	log.InitializeLogger(logger, zapConfigs)
+}
+
```

Comment:
This is never called, the `StartAllServicesWithLogger()` is called from the k8shim entrypoint we never just want to call this as part of the entry

Reply:
It gets called in the updated shim code when the configuration changes, so it's needed here.

Reply:
Per offline discussions, reverting all logging changes.